srcpkgs/KataGo-OpenCL/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'KataGo-OpenCL'
+pkgname=KataGo-OpenCL
+version=1.11.0
+revision=1
+#archs="i686 x86_64"
+wrksrc="KataGo-${version}"
+#create_wrksrc=yes
+build_wrksrc=cpp
+build_style=cmake
+configure_args="-DUSE_BACKEND=OPENCL -DNO_GIT_REVISION=1"
+
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	configure_args+=" -DCMAKE_CXX_STANDARD_LIBRARIES=-latomic"
+	makedepends+=" libatomic-devel"
```

Comment:
`if` blocks usually go after variables block. In this case it is not only matter of style: `makedepends` is overwritten later on. Please move below `checksum`.